lms/static/sass/shared/_header.scss

```diff
@@ -264,6 +264,10 @@ header.global {
         }
       }
     }
+
+    @media print {
```

Comment:
Yes, see the PDF example I linked in the introduction to this PR:
![screen shot 2015-04-17 at 3 38 29 pm](https://cloud.githubusercontent.com/assets/6232546/7209577/e31a2928-e517-11e4-8457-0f97449e8419.png)